truetype/truetype.go

```diff
@@ -306,6 +317,65 @@ func (f *Font) parseMaxp() error {
 	return nil
 }
 
+func (f *Font) parseName() error {
+	if len(f.name) < 18 {
+		return FormatError("name data too short")
+	}
+	if u16(f.name, 0) != 0 {
+		return FormatError(fmt.Sprintf("invalid format specifier: %d", u16(f.name, 0)))
+	}
+	numNames := int(u16(f.name, 2))
+	stringOffset := u16(f.name, 4)
+	font := &nr{}
```

Comment:
"font" seems an odd variable name, or at least a not very informative one. Lots of things in this file are related to fonts.